### DISCUSSION TITLE
How can I change the Ports and what is the settings in the Client-Software

### DISCUSSION THREAD

**Discussion by Wolf6660** [rustdesk/rustdesk#2121] [Q&A]
Hi,

I hope you understand my bad english.

I want to create a RustDesk server on my Synology, but I want to change the ports. I want to control Mac and Windows PCs with the client software.

If I understand the manual correctly, I need ports 21115, 21116 and 21117.

How can I change the port?

for example:
21115 -> 34567
21116 -> 45678
21117 -> 56789

I installed Docker with the original ports and forwarded internal 21115 to 34567 and so on in my router.

What are the correct client settings?

Is this the correct procedure, or how should I best adjust it?

thanks

Daniel

**Comment by keyamasabaya** [comment]
Is there a solution for this question ?
